.github/workflows/documentation.yml

```diff
@@ -13,8 +13,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: '14'
-      - name: Upgrade NPM
+          node-version: '>=16.14'
         run: npm install -g npm
         # see https://github.com/bahmutov/npm-install/issues/103#issuecomment-931226602
```

Comment:
```suggestion
```

Do we still need this, now that we're specifying Node 16?

Reply:
I'll remove it!